### PR TITLE
DOC: gpu contribution instructions

### PR DESCRIPTION
**Pull request by seanpmorgan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,7 +88,26 @@ docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-c
 
 bazel test -c opt -k \
 --test_timeout 300,450,1200,3600 \
---test_output=errors \
+--test_output=all \
+//tensorflow_addons/...
+```
+
+### Locally Testing GPU
```
Comment (WindQAQ) [inline CONTRIBUTING.md]: Maybe using #### which is same with above?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -88,7 +88,26 @@ docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-c
...
+```bash
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:custom-op-gpu make gpu-unit-test
+```
+
+or run manually:
+
+```bash
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:custom-op-gpu /bin/bash
+export TF_NEED_CUDA=1
```
Comment (facaiy) [inline CONTRIBUTING.md]: TF_NEED_CUDA has existed in the custom-op-gpu image

```bash
$docker run --rm -it tensorflow/tensorflow:custom-op-gpu bash -c 'echo $TF_NEED_CUDA'
1
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (seanpmorgan) [reply CONTRIBUTING.md]: Good catch thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (seanpmorgan) [reply CONTRIBUTING.md]: Thanks! Removed.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -88,7 +88,25 @@ docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-c
...
 bazel test -c opt -k \
 --test_timeout 300,450,1200,3600 \
---test_output=errors \
+--test_output=all \
+//tensorflow_addons/...
+```
+
+#### Locally Testing GPU
+```bash
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:custom-op-gpu make gpu-unit-test
```
Comment (facaiy) [inline CONTRIBUTING.md]: Perhaps it's recommend to use `bash tools/run_docker.sh -d gpu -c 'make gpu-unit-test'` for brevity？

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -73,7 +73,7 @@ Please see our [Style Guide](STYLE_GUIDE.md) for more details.
 Nighly CI tests are ran and results can be found on the central README. To 
 subscribe for alerts please join the [addons-testing mailing list](https://groups.google.com/a/tensorflow.org/forum/#!forum/addons-testing).
 
-#### Locally Testing
+#### Locally Testing CPU
 
 ```bash
 docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-custom-op make unit-test
```
Comment (facaiy) [inline CONTRIBUTING.md]: `bash tools/run_docker.sh -c 'make unit-test'`

Comment (facaiy) [reply CONTRIBUTING.md]: And we also can revise Development Environment section above:
```
Try these commands below:
* Format code automatically:  bash tools/run_docker.sh -c 'make code-format'
... ...
```

Comment (seanpmorgan) [reply CONTRIBUTING.md]: Good suggestions... thanks! Updated.

**Comment by facaiy** [comment]
@WindQAQ Tzu-Wei, please merge the PR if you think it good :-)

**Comment by WindQAQ** [comment]
Thanks!

**Pull request merged**